Setup/docker-compose_dev.yml

```diff
@@ -98,12 +97,37 @@ services:
           memory: 1GB
           cpus: "1"
     environment:
-        - "CONSUL_BIND_INTERFACE=eth2"
+        - "CONSUL_BIND_INTERFACE=eth0"
         - "NUMBER_OF_EXPECTED=1"
     logging:
       driver: syslog
       options:
         syslog-address: udp://${SYS_LOG_HOST}:5014
+  consul:
+      image: securebrowsing/shield-configuration:multi
+      networks:
+           - shield
+      ports:
+          - "8500:8500"
+      deploy:
+        mode: replicated
+        replicas: 1
+        update_config:
+            parallelism: 1
+            failure_action: rollback
+        restart_policy:
+            condition: any
+        resources:
+          limits:
+            memory: 1GB
+            cpus: "1"
+      environment:
+          - "CONSUL_BIND_INTERFACE=eth2"
+          - "RUN_AGENT=yes"
```

Comment:
What Run_Agent means?
Please add a comment
Does RUN_AGENT is yes by default?
if this is the case, please comment it